metadata-ingestion/src/datahub/ingestion/source/glue_schema_registry.py

```diff
@@ -0,0 +1,404 @@
+import json
+import logging
+from dataclasses import dataclass
+from hashlib import md5
+from typing import Any, List, Optional, Set, Tuple
+import boto3
+import jsonref
+from confluent_kafka.schema_registry.schema_registry_client import (
+    RegisteredSchema,
+    Schema,
+    SchemaReference
+
```

Comment:
Not clear why we would need confluent_kafka imports here.
The idea to support additional schema registry is to provide implementation for following method for new schema registry

```
 def get_schema_metadata(
        self, topic: str, platform_urn: str
    ) -> Optional[SchemaMetadata]:
```
Assuming that glue client and aws schema registry client has all methods, classes for interfacing with schema, these imports should not be necessary.